import_tracker/rest.py

```diff
@@ -1,22 +1,77 @@
 # -*- coding: utf-8 -*-
 from girder.api import access
+from girder.utility import path, model_importer
 from girder.api.describe import Description, autoDescribeRoute
 from girder.constants import SortDir
+from girder.api.rest import boundHandler
+from girder.models.assetstore import Assetstore
+from girder.models import getDbConnection
+
+
 from .models import AssetstoreImport
 
 from bson.objectid import ObjectId
 
 
+def processCursor(cursor, user):
+    lookedupAssetstores = {}
+    results = list(cursor)
+    db = getDbConnection()
+
+    for row in results:
+        if row['assetstoreId'] not in lookedupAssetstores:
+            assetstore = list(Assetstore().find({'_id': row['assetstoreId']}))
+            lookedupAssetstores[row['assetstoreId']] = assetstore[0]['name']
+
+        row['_assetstoreName'] = lookedupAssetstores[row['assetstoreId']]
+        model = model_importer.ModelImporter.model(row['params']['destinationType'])
+        doc = model.load(row['params']['destinationId'], user=user)
+        if(db.girder[row['params']['destinationType']].find(
+                {'_id': ObjectId(row['params']['destinationId'])}
+                ).count()):
```

Comment:
This can just be `if doc:`, since the doc loaded on line 28 will be None if it doesn't exist.

Reply:
fixed in c47715346b33d278e67eff0fd063491c611f0eea